<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinMaxThresholdWidget</class>
 <widget class="QWidget" name="MinMaxThresholdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intensity Controller</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="minMaxLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="minLabel">
           <property name="text">
            <string>min</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="minSpinBox"/>
         </item>
         <item row="0" column="2">
          <widget class="QSlider" name="minSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="maxLabel">
           <property name="text">
            <string>max</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="maxSpinBox"/>
         </item>
         <item row="1" column="2">
          <widget class="QSlider" name="maxSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="thresholdLabel">
           <property name="text">
            <string>threshold</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="thresholdSpinBox"/>
         </item>
         <item row="2" column="2">
          <widget class="QSlider" name="thresholdSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="morePushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>More...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="limitsGroupBox">
         <property name="title">
          <string>Limits</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="limitsGridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="minLimitLabel">
              <property name="text">
               <string>min</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="minLimitSpinBox">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="maxLimitLabel">
              <property name="text">
               <string>max</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="maxLimitSpinBox">
              <property name="minimum">
               <double>-1000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
